ruby/ruby_releases/ruby_benchmarks/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Just use build for all it doesn't matter. To be able to run the script, it already assumes that we are in the right directory